pkg/service_quotas/service_quotas.go

```diff
@@ -100,10 +102,6 @@ type QuotasInterface interface {
 // or returns an error. Note that the ServiceQuotas will only return
 // usage and quotas for the service quotas with implemented usage checks
 func NewServiceQuotas(region, profile string) (QuotasInterface, error) {
-	if !isValidRegion(region) {
-		return nil, errors.Wrapf(ErrInvalidRegion, "failed to create ServiceQuotas: %w")
-	}
-
```

Comment:
That check is still useful to validate the region.

Reply:
Right... i thought that the aws sdk would do some sort of validation for regions but apparently it doesn't. I'll add this back in.